vernac/comSearch.mli

```diff
@@ -10,5 +10,11 @@
 
 (* Interpretation of search commands *)
 
+val interp_search_request :
+  Environ.env ->
+  Evd.evar_map ->
+  bool * Vernacexpr.search_request ->
+  bool * Search.glob_search_request
```

Comment:
What are the bools?

Reply:
I don't know. AFAICT, they were introduced in https://github.com/coq/coq/commit/96333c1bf27f33c15e2475c11c7bfefe87d3a6e1

Reply:
seems to be about if the condition is positive or negative